source/simulator/simulator_access.cc

```diff
@@ -364,6 +364,24 @@ namespace aspect
     return simulator->traction_boundary_conditions;
   }
 
+   template <int dim>
+   const std::set<types::boundary_id >
+   SimulatorAccess<dim>::get_traction_boundary_indicators () const
+   {
+     std::map< types::boundary_id, std::pair<std::string, std::string> > map_indicators
+       = simulator->parameters.prescribed_traction_boundary_indicators;
+     std::map< types::boundary_id, std::pair<std::string, std::string> >::iterator it;
+     std::map< types::boundary_id, std::pair<std::string, std::string> >::iterator begin = map_indicators.begin();
+     std::map< types::boundary_id, std::pair<std::string, std::string> >::iterator end = map_indicators.end();
```

Comment:
It will make this easier to read if you just put the two lines above into the for-loop below, i.e. 
`for(it = map_indicators.begin(); it != map_indicators.end(), ++it)`